### ISSUE TITLE
Add support for https in Nginx proxy

### ISSUE THREAD

**Issue by samber** [RobustPerception/demo_prometheus_ansible#5]
Hi,

For now, Prometheus console is only available in **http**, on port 9090.

I would like to use your Prometheus instance as a backend for the demonstrations of 2 JS library I'm working on:

- https://github.com/samber/prometheus-query-js
- https://github.com/samber/chartjs-plugin-datasource-prometheus

Github pages are hosted with HTTPS URLs. Browser are not allowed to send requests to insecure HTTP endpoint.

Are you willing to add TLS to this nginx proxy ? Do you have a robustperception.io wildcard certificate ? Do you prefer LetsEncrypt ?

**Comment by brian-brazil** [comment]
The challenge here is that making it work properly work over https means setting the external url, which would break http.

**Comment by samber** [comment]
Yes. But can we expose one more port on your server ?

**Comment by brian-brazil** [comment]
Yes, there's apache running. But Prometheus's web handling is only designed to work on a single URL.
